pkg/config/cos.go

```diff
@@ -130,10 +130,33 @@ func ConvertToCOS(config *HarvesterConfig) (*yipSchema.YipConfig, error) {
 		}
 	}
 
+	// Add Setting CR preconfigs
+	fsStages := make([]yipSchema.Stage, 0)
+
+	if len(config.SystemSettings) != 0 {
+		settingCrs, err := render("settings.yaml", config)
+		if err != nil {
+			return nil, err
+		}
+
+		fsStages = append(fsStages, yipSchema.Stage{
+			Files: []yipSchema.File{
+				{
+					Path:        "/var/lib/rancher/rke2/server/manifests/settings.yaml",
```

Comment:
```suggestion
					Path:        "/var/lib/rancher/rke2/server/manifests/harvester-settings.yaml",
```

It would be nice to add a prefix. The directory is a bit crowded.

```
node1:~ # ls /var/lib/rancher/rke2/server/manifests/
harvester-config.yaml      monitoring-crd.yaml        monitoring.yaml            rke2-canal-config.yaml     rke2-coredns.yaml          rke2-multus.yaml
harvester.yaml             monitoring-dashboard.yaml  rancher/                   rke2-canal.yaml            rke2-metrics-server.yaml
```

I also consider moving all harvester things to a new directory `/var/lib/rancher/rke2/server/manifests/harvester`.
But considering we might switch back to ManagedChart for the upgrade, we can skip it now.

Reply:
Ok, I'll rename it to `harvester-settings.yaml`.